zproject/urls.py

```diff
@@ -285,6 +286,10 @@
         PATCH=update_realm_custom_profile_field,
         DELETE=delete_realm_custom_profile_field,
     ),
+    rest_path(
+        "realm/profile_fields/display_in_profile_popover/<int:field_id>",
+        PATCH=toggle_display_in_profile_popover_realm_custom_profile_field,
+    ),
```

Comment:
I don't think it makes sense to define a new endpoint -- we should just use the existing `update_realm_custom_profile_field` function / endpoint. 

From a design standpoint, that endpoint is for updating properties of a custom profile field, and this is just one of those properties.

Reply:
Yeah, this thing was in my mind while defining a new endpoint here, and yes I am already using `update_realm_custom_profile_field` for updating custom profile fields, this new endpoint I defined is just for toggle checkboxes for the `display_in_profile_popover` field in the settings UI.